qemu/tests/ioeventfd.py

```diff
@@ -175,8 +175,20 @@ def _compare_ioeventfds():
     ioeventfds = (params['orig_ioeventfd'], params['new_ioeventfd'])
     for ioeventfd in ioeventfds:
         dev_id = _set_ioeventfd_options()
+        # Disable iothread when ioeventfd=off
+        if ioeventfd == params['orig_ioeventfd'] and params.get(
+                "iothread_scheme"):
```

Comment:
I think :
```
        if ioeventfd == 'ioeventfd=off' and params.get(
                "iothread_scheme"):
```
is more accurate since if add a new case that it assign the `orig_ioeventfd = 'ioeventfd=on' ` later.
so i think compare whether the value of paramter is `'ioeventfd=on' `is better.

Reply:
updated.